infrastructure/variables.tf

```diff
@@ -203,3 +203,28 @@ variable "create_openshift_instances" {
   type    = bool
   default = true
 }
+
+variable "reuse_tags" {
+  type        = bool
+  description = "Use existing tag namespace and defined tags when tagging OCI resources. Tag namespace and defined tags are preserved when the stack is destroyed. Warning: Stack creation will fail if specifed tag namesapce and defined tags do not exist as specified. Create stack with this set to false to create tagging resources that are destroyed when the cluster is, or follow this link for terraform to create tagging resources seperately. It's recommended you do not change this flag after cluster creation to preserve terraform state consistency."
+  default     = false
+}
+
+variable "tag_namespace_name" {
+  type        = string
+  description = "Name of tag namespace to create or use for tagging OCI resources. Defaults to \"openshift-{cluster_name}\""
+  default     = ""
+}
+
+variable "tag_namespace_compartment_ocid" {
```

Comment:
Can we include these variables in the schema.yaml?
For example, if we add the compartment_ocid into the schema,
It will allow the user to select the compartment from a drop-down menu instead of typing the full ocid
ref:
https://github.com/oracle-quickstart/oci-openshift/blob/beta-v1/infrastructure/schema.yaml#L76

Reply:
Maybe within the same section of OpenShift Cluster Configuration?
https://github.com/oracle-quickstart/oci-openshift/blob/beta-v1/infrastructure/schema.yaml#L14

Reply:
I can work on that. Maybe it would also make more sense to have a Tagging Resources section instead of adding it to an existing one